README.md

```diff
@@ -172,6 +173,25 @@ plugins = [
 Once the changes have been committed to the main branch, the new image will automatically
 be generated.
 
+## The Jupyter image
+
+In addition to the Docker image for running the oasis, this repository also builds a custom NORTH image for running a jupyter hub with the installed plugins.
+This image has been added to the [`configs/nomad.yaml`](configs/nomad.yaml) during the initialization of this repository and should therefore already be available in your Oasis under "Analyze / NOMAD Remote Tools Hub / jupyter"
```

Comment:
Can you add a hint to pre pull the image?

Reply:
Ah, yes good point!